smartseq2_single_sample/ss2_checker.wdl

```diff
@@ -0,0 +1,15 @@
+task SmartSeq2Checker {
+  File rna_metrics
+
+  String expected_rna_metrics_hash
+
+  command <<<
+    rna_metrics_hash=$(tail -n +6 ${rna_metrics} | md5sum | awk '{print $1}')
+    
+    if [ "$rna_metrics_hash" != "${expected_rna_metrics_hash}" ]; then
```

Comment:
should it be `"${rna_metrics_hash}"` instead of `"$rna_metrics_hash"` ?

Reply:
I think it would be the same, but I prefer to use different syntax for bash variables and WDL variables when I can. However, has everyone else settled on using braces around bash variables?

Reply:
Yeah, it should be the same. To add `{}` just just to make variables in wdl consistent, pure cosmetics effect.